workflows/Discord_Message_Notify.yml

```
@@ -0,0 +1,30 @@
+name: Discord Message Notify
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  notify:
```

We should also make sure this only runs on this repo, and not on forks. It will fail on forks anyway, it's more of a safety that we don't use up their github action time for no good reason...

```suggestion
  notify:
    if: github.repository == 'AlternativeFFFF/Alt-F4'